src/ui/globals/types.ts

```diff
@@ -22,6 +22,9 @@ enum OperationType {
   SCAN_CONNECTION = "scanConnection",
   ADD_CREDENTIAL = "addCredential",
   RECEIVE_CONNECTION = "receiveConnection",
+  CREATE_IDENTIFIER_CONNECT_WALLET = "createIdentifierConnectWallet",
+  SCAN_WALLET_CONNECTION = "scanWalletConnection",
+  PASTE_MKID_CONNECT_WALLET = "pasteMkidConnectWallet",
```

Comment:
I don't think we need these operation types.

We will discuss again after the refactoring 👍